### ISSUE TITLE
Unable to override Login page

### ISSUE THREAD

**Issue by hishamco** [OrchardCMS/OrchardCore#17046]
I have an admin theme in Orchard Core Contrib (OCC) that overrides the login previously without any issue https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/blob/main/src/OrchardCoreContrib.Themes.Admin/Views/OrchardCore.Users/Account/Login.cshtml after upgrading OC to `2.0.0` the UI is broken, I fixed almost thing but the issue I can't override the login form. I tried to add `LoginForm.Edit.cshtml` and `LoginFormCredentials.cshtml` with no luck

/cc @MikeAlhayek

**Comment by MikeAlhayek** [comment]
@hishamco what does broken means?

To override the login form you'll need to override the view `LoginForm.Edit.cshtml`

If you want to override the Username and the Password you'll override the `LoginFormCredentials.cshtml` view.

**Comment by hishamco** [comment]
> @hishamco what does broken means?

Not displayed properly

I already override it with no luck, I'm unable to change the login button style nor any text in the form

**Comment by hishamco** [comment]
I just override the views, or did I miss something else?

**Comment by MikeAlhayek** [comment]
If you override the views correctly, then it should have worked. If you share the name of the view you overrode and the code you used in each view, I can try to see what you could be doing wrong.

**Comment by hishamco** [comment]
I refer to the code above and the proposed views too

**Comment by MikeAlhayek** [comment]
The view [OrchardCoreContrib/OrchardCoreContrib.Modules/blob/main/src/OrchardCoreContrib.Themes.Admin/Views/OrchardCore.Users/Account/Login.cshtml](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/blob/main/src/OrchardCoreContrib.Themes.Admin/Views/OrchardCore.Users/Account/Login.cshtml) seems to be identical to OC's version. what changes were you looking to do?

**Comment by hishamco** [comment]
Style the login button for instance

**Comment by MikeAlhayek** [comment]
Yes for that you'll need to override the Login.cshtml view. Maybe you're not placing that view in the right path. Is it in a theme? Where is it placed?

**Comment by MikeAlhayek** [comment]
Yes LoginForm.Edit.cshtml is what you need to override if you want to change the form section of the Login.cshtml

**Comment by hishamco** [comment]
I already mentioned [here](https://github.com/OrchardCMS/OrchardCore/issues/17046#issue-2684427053) that I already override it with no luck

**Comment by dturska** [comment]
Do you have the [Use site theme for login page - Requires an active site theme.]  checked?
![image](https://github.com/user-attachments/assets/0a8e0860-8384-406a-9036-347b97fb8810)

**Comment by hishamco** [comment]
It's an admin theme

**Comment by ns8482e** [comment]
@hishamco which version? I have no issue with 2.0.2

**Comment by hishamco** [comment]
`2.0.0`

**Comment by devlife** [comment]
> It's an admin theme

Will it work with an Admin theme or only a front end theme?

**Comment by sebastienros** [comment]
@hishamco you wrote

> If you look at the link above the login is placed in the theme inside OrchardCore.Users\Account

But `LoginForm.Edit.cshtml` is a shape, so it won't be found there. Have you tried to put it in a `Views` folder?
